src/stripepy/IO.py

```diff
@@ -25,52 +24,35 @@ class ANSI:
     ENDC = "\033[0m"
 
 
-# TODO: Rename function
-# The current name gives the impression that calling this function will list existing folders
-# while in reality the function **generate** a list of folder (names)
-def list_folders_for_plots(path: os.PathLike) -> List[pathlib.Path]:
-    # TODO: should this function check that path exists and is an existing folder?
+def remove_and_create_folder(path: pathlib.Path, force: bool):
     path = pathlib.Path(path)
-    return [
-        path,
-        path / "1_preprocessing",
-        path / "2_TDA",
-        path / "3_shape_analysis",
-        path / "4_biological_analysis",
-        path / "3_shape_analysis" / "local_pseudodistributions",
-    ]
-
-
-def remove_and_create_folder(path):
 
     # Deleting folders:
-    if os.path.exists(path):
-
-        # TODO: in release, ask to confirm deletion...
-        # confirm = input("The output folder already exists. Do you want to delete it? (y/n) ")
-        confirm = "y"
-
-        if confirm.lower() == "y":
-            # Remove the folder and its contents
-            shutil.rmtree(path)
-            print("Output folder and its contents have been deleted.")
+    if path.exists():
+        if not force:
+            raise RuntimeError(f"refusing to overwrite file {path}. Pass --force to overwrite.")
```

Comment:
I think it is better to rewrite the error message to make it clear that it is a folder that will be deleted when `--force` is given

Reply:
Good point!